src/core/Table/hooks/useSubRowFiltering.tsx

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { Hooks, IdType, Row, TableInstance } from 'react-table';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import * as defaultFilterTypes from '../../../../node_modules/react-table/src/filterTypes.js';
```

Comment:
What is this atrocious import path?

Reply:
To get default `reat-table` filter functions. https://github.com/tannerlinsley/react-table/blob/master/src/filterTypes.js

Reply:
Cant import from `react-table/src/filterTypes`?

Reply:
Apparently I can 😮 I tried previously and intellisense didn't show anything.

Reply:
Are you also able to remove `* as`?

Reply:
Apparently no.